app/models/driver.rb

```diff
@@ -0,0 +1,8 @@
+class Driver < ApplicationRecord
+  validates :name, presence: true
+  validates :license_number, presence: true
+  validates :mobile_number, presence: true, phone: true
+  validates :email_address, presence: true, email: true
+  validates :age, presence: true
```

Comment:
For `age` it is also good to add validation on numericality and range
https://guides.rubyonrails.org/active_record_validations.html#numericality